### PR TITLE
Feat/thanos sepolia deposit history

### PR DESCRIPTION
**Pull request by black02430horse**
## Summary

- Display history for thanos sepolia deposit transactions.
- Cache Txs for better performance
- Real time update of pending Txs

## Vercel link
https://deposit-history-tokamak-bridge.vercel.app/
image

## Checklist

- [x] Provided a screenshot (only if change is visible in UI)
- [ ] Provided steps for playback (only if possible).
- [x] Provided a change scope in summary
- [x] Local build has passed

### CODE REVIEW

File: src/staging/components/new-confirm/components/core/other/index.tsx [hunk 1]
```diff
@@ -50,11 +50,12 @@ export default function DepositWithdrawConfirmModal() {
   const { depositWithdrawConfirmModal, onCloseDepositWithdrawConfirmModal } =
     useDepositWithdrawConfirmModal();
   const transactionData = depositWithdrawConfirmModal.transaction;
-
   const { address } = useAccount();
   const { onClick } = useCallBridgeSwapAction();
   const { totalGasCost, gasCostUS } = useGasFee();
   const networkChainId = transactionData?.outNetwork || THANOS_SEPOLIA_CHAIN_ID;
+  const inNetworkCHainId =
```
Comment (SonYoungsung) [inline src/staging/components/new-confirm/components/core/other/index.tsx:57]: CHainId seems to be a typo; it should be ChainId.

File: src/staging/constants/transactionTime.ts [hunk 2]
```diff
@@ -14,3 +16,9 @@ export const TRANSACTION_CONSTANTS = {
     RETURN_LIQUIDITY: 300,
   },
 };
+
```
Comment (SonYoungsung) [inline src/staging/constants/transactionTime.ts:19]: actually i already told about it to Robert. I don't think it's a good idea that we set constant time value as minute. I will have a plan to change it to seconds when he's back.

**Review by SonYoungsung — COMMENTED** [review]
There seem to be two issues. First, the Deposit history does not appear in the history tab when using my test account. Second, the Withdraw history is not displayed correctly, and it seems that the components related to Rollup and Finalize are not functioning properly.

Additionally, you attempted memory caching through Recoil, but it might be more meaningful to utilize the browser's storage for caching as well. This isn't urgent, so it would be good to try this when there’s an opportunity later.

File: src/staging/components/new-confirm/components/core/other/index.tsx (hunk 1 above)
Comment (black02430horse) [reply src/staging/components/new-confirm/components/core/other/index.tsx:57]: Good catch! fixed.

File: src/staging/constants/transactionTime.ts (hunk 2 above)
Comment (black02430horse) [reply src/staging/constants/transactionTime.ts:19]: Sounds good! I prefer not to fix legacy code unless it's critical, as we might encounter many conflicts from working on different branches simultaneously. Anyway Let's leave those fixes for Robert when he's back. :)
plus, we might need to polish the entire codebase and increase test coverage at some point when we are not so busy.

**Pull request merged**